### ISSUE TITLE
Rotations of instances within scaled groups

### ISSUE THREAD

**Issue by mbc** [mitmedialab/para#130]
When a group is scaled, and then an instance of the group is rotated, the result seems strange (to me) because the rotation is performed prior to the scaling, i.e. the rotation as done by the user is not orthogonal (so the shape is not preserved). Is this the right behavior? To me it would be more natural if the instance, as scaled as part of the group, is what's rotated.

Video here: [https://drive.google.com/file/d/0B0gX7qtFYmLocEF4bTdZMlpyZnc/view?usp=sharing](https://drive.google.com/file/d/0B0gX7qtFYmLocEF4bTdZMlpyZnc/view?usp=sharing)

**Comment by pixelmaid** [comment]
I agree this is counter- intuitive and that the behavior you propose is more desirable. I'm concerned however that changing this would require a major modification to the transformation/rendering architecture, and I'm not sure I want to completely overhaul it as there are more pressing issues.

You can take a look at the transformation order of operations and see if there's an obvious fix, but if it's going to require major changes, I don't think we should go there yet..
 https://github.com/mitmedialab/para/blob/dev/js/src/models/data/Instance.js#L1576
